.spi.yml

```diff
@@ -0,0 +1,5 @@
+version: 1
+builder:
+  configs:
+    - platform: ios
+      documentation_targets: [Braintree]
```

Comment:
Looking at some examples on github it seems like we may need to list each target independently? I was looking at [swift collections](https://github.com/apple/swift-collections/blob/939cfd25234472b4dc91c3caeab304d15bca9a73/.spi.yml#L4) as an example. Though it looks like maybe this just works? Not totally sure. This is super neat though!

Reply:
Good find! I was thinking this too, but I didn't find a good example of an SDK that used multiple targets. Looking at [Swift Collections Package.swift](https://github.com/apple/swift-collections/blob/939cfd25234472b4dc91c3caeab304d15bca9a73/Package.swift), this looks similar to how our project is separated out. Will update

Reply:
Done - f264f9c5c3da9a6fe10315bfa90086da1324f76e